aerogear/aerogear-ios-http
# Convert AeroGearHttp Test scheme to Integration Tests scheme

this PR adds an 'IntegrationTests' target that enables tests to be run within the context of the emulator. Details of why this was required can be found on our JIRA [AGIOS-330](https://issues.jboss.org/browse/AGIOS-330)

To test:
After running ```pod install``` and opening the workspace, choose ```IntegrationTest``` schema, and then choose  ```Product->Test```. Tests should run and succeed.

@corinnekrych @danbev mind to have a look?
